fine_tune.py

```diff
@@ -311,13 +325,13 @@ def fn_recursive_set_mem_eff(module: torch.nn.Module):
 
         for step, batch in enumerate(train_dataloader):
             current_step.value = global_step
-            with accelerator.accumulate(training_models[0]):  # 複数モデルに対応していない模様だがとりあえずこうしておく
+            with accelerator.accumulate(*training_models):
                 with torch.no_grad():
                     if "latents" in batch and batch["latents"] is not None:
                         latents = batch["latents"].to(accelerator.device)  # .to(dtype=weight_dtype)
```

Comment:
Why is this commented out? It breaks mixed-precision training in deepspeed.
I added it back and bf16 works again (even with no_half_vae=true)

```suggestion
                        latents = batch["latents"].to(accelerator.device, dtype=weight_dtype)
```